### PR TITLE
Add build/publish of Windows Docker Studio to release pipeline

### PR DESCRIPTION
**Pull request by smacfarlane**
This takes code from components/hab-bintray-publish and integrates it into the release pipeline.

Signed-off-by: Scott Macfarlane <smacfarlane@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: .buildkite/scripts/publish_docker_studio.ps1 [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    # docker push "${docker_image}:$docker_image_version"
+    # Write-Host "Pushing ${docker_image}:$docker_image_short_version tag for $docker_image_version"
+    # docker push "${docker_image}:$docker_image_short_version"
+    # Write-Host "Pushing latest tag for $docker_image_version"
+    # docker push "${docker_image}:latest"
+  # }
+  # finally {
+      # Remove-Item $HOME/.docker/config.json -Recurse -Force -ErrorAction SilentlyContinue
+  # }
+}
```
Comment (mwrock) [inline .buildkite/scripts/publish_docker_studio.ps1]: commented code looks ok to me

**Comment by mwrock** [comment]
Oh we should also delete the scripts in `components/studio` and `bintray-publish`

**Comment by smacfarlane** [comment]
> Oh we should also delete the scripts in `components/studio` and `bintray-publish`

I looked at doing this, but we need to do some refactoring of the bintray-publish plan, and pipeline steps first.  I've filed #6179 to track that work

**Pull request merged**